mmv1/templates/terraform/custom_flatten/artifactregistry_rr_disable_upstream_validation.go.erb

```diff
@@ -0,0 +1,3 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+  return false
```

Comment:
Why do we need to return `false` specifically? We'll need to return whatever read from state, so probably `d.get("remote_repository_config.0.disable_upstream_validation")`?

Reply:
> Why do we need to return `false` specifically? We'll need to return whatever read from state, so probably `d.get("remote_repository_config.0.disable_upstream_validation")`?

Ah my apologies, I thought I was supposed to set this to what the API returns (will never return true), not what the request contains.

After updating to what you suggested I'm running into a new error I haven't seen before about an attribute missing after import. Do you know how we can skip this verification for this test? As discussed before the registry will always return false no matter what this field is set to in the request. Here's the full error below:

```
=== CONT  TestAccArtifactRegistryRepository_artifactRegistryRepositoryRemoteCustomExample
    vcr_utils.go:152: Step 2/2 error running import: ImportStateVerify attributes not equivalent. Difference is shown below. The - symbol indicates attributes missing after import.

          map[string]string{
        -       "remote_repository_config.0.disable_upstream_validation": "true",
        +       "remote_repository_config.0.disable_upstream_validation": "false",
          }
```

Reply:
You'll need to add the field to `ignore_read_extra` for any examples since we read the field from state. See more detail at https://googlecloudplatform.github.io/magic-modules/develop/permadiff/#ignore_read

Reply:
Ahhhhh damn I was using the wrong value for ignore_read_extra (was just using `disable_upstream_validation` instead of the full path.

Thank you so so much @shuyama1!! I really appreciate the help working through this :) The tests should pass now, let me know if you'd like me to change anything else with this in order to merge